src/components/app/details/appDetails/utils.tsx

```diff
@@ -213,27 +213,56 @@ export function getCalendarValue(startDateStr: string, endDateStr: string): stri
     return str;
 }
 
-export function getIframeSrc(appId: string | number, envId: string | number, environmentName: string, chartName: ChartTypes, newPodHash: string, calendarInputs, tab: AppMetricsTabType, isLegendRequired: boolean, statusCode?: string):string {
+export function isK8sVersion115AndBelow(k8sVersion: string): boolean {
+    let target = [1, 15, 0];
+    let version: string[] = []
+    try {
+        version = (k8sVersion.split("v")[1]).split(".");
+    } catch (error) {
+        
```

Comment:
what are we doing with error? if we have error then we dont know... so we should make an assumption and return true or false. There is no reason to move forward